site/content/en/docs/manual/advanced/formats/format-camvid.md

```diff
@@ -22,8 +22,9 @@ taskname.zip/
 |   └── image2.png
 └── <any_subset_name>.txt
 
-# labelmap.txt
+# label_colors.txt
 # color (RGB) label
+# if you do not enter a color (RGB) value, it will be set automatically.
```

Comment:
Could you please add an example of this? I'm not sure it's clear now how this can be done correctly.

Reply:
@zhiltsov-max Hello. Sure! I recorded the video about this.

I recorded and uploaded a video about automatically applying colors in cvat(camvid format) when we did not enter color values ​​for each class in `label_colors.txt`. Check the video and confirm that the color value is set automatically if the color value is not entered.

The video size is so big, so I uploaded by google drive
recorded video link: https://drive.google.com/file/d/1LxSlVMJJQPmfNR-8olNhQ3vTHwk6pSgQ/view?usp=drive_link

Reply:
@zhiltsov-max 

The [CVAT CamVid documentation](https://opencv.github.io/cvat/docs/manual/advanced/formats/format-camvid/) explains it as label_colors.txt must need color values, but if you check the [example video](https://drive.google.com/file/d/1LxSlVMJJQPmfNR-8olNhQ3vTHwk6pSgQ/view?usp=drive_link), you can see that it can be used in CVAT by automatically setting color values ​​even if there is no color value for the class in label_colors.txt.

- CVAT CamVid documentation explains
<img width="731" alt="camvid_docs" src="https://github.com/opencv/cvat/assets/130522722/807ee326-05bd-4e9d-81d4-79e739fc187b">

- You can see in the example video that the color value is loaded automatically even if there is no color value in label_colors.txt.
<img width="673" alt="camvid_no_color" src="https://github.com/opencv/cvat/assets/130522722/60bef4ec-c3ed-4843-9809-da621ae242ae">

Reply:
Thanks for the video. I'm just asking to add this into the PR, because other users can be confused about this line without an example.

Reply:
@zhiltsov-max 
Do you want to add this video to this PR, rather than leaving it as a comment?
Feel free to add an example video to that PR :)

Reply:
No, just a couple of lines with the label file example without color information.